recipes/pyusb/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyusb" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - libusb  # [unix]
```

Comment:
Is this selector needed? Seems we have the Windows package as well.

Reply:
Should add if we are able to drop the selector here, it would be good to make this `noarch: python` as well.

Reply:
I also think we can get rid off the selectors here
From ``pyusb``:
> Remember that you need libusb (1.0 or 0.1) or OpenUSB running on your system. For Windows users, libusb 0.1 is provided through libusb-win32 package

Reply:
So, it seems to be a dependency of ``pyusb`` as well

Reply:
Went ahead and added `noarch: python`. Hope that is ok. 🙂